test/test_serialization.h

```diff
@@ -739,6 +739,30 @@ TEST(serialization, serialize_softplus) {
   EXPECT_FLOAT_EQ(net.at<softplus_layer>(0).threshold_value(), float_t(20));
 }
 
+TEST(serialization, serialize_softsign) {
+  network<sequential> net;
+
+  std::string json = R"(
+    {
+        "nodes": [
+            {
+                "type": "softsign",
+                "in_size" : {
+                    "width": 10,
+                    "height" : 10,
+                    "depth" : 3
+                }
+            }
+        ]
+    }
+    )";
+
+  net.from_json(json);
```

Comment:
There is an error at this line but I don't knoe how to fix it (Possibly a problem with json?). Any suggestion @karandesai-96 @Randl ?